models/rust/src/3.0/files/examples/specification/system_pricing_plans-1.json

```diff
@@ -0,0 +1,46 @@
+{
+    "last_updated": "2023-07-17T13:34:13+02:00",
+    "ttl": 0,
+    "version": "3.0",
+    "data": {
+        "plans": [
+            {
+                "plan_id": "plan2",
+                "name": [
+                    {
+                        "text": "One-Way",
+                        "language": "en"
+                    }
+                ],
+                "currency": "USD",
+                "price": 2.00,
+                "reservation_price_per_min": 0.15,
```

Comment:
This field was removed from v3.0 and added to v3.1-RC ([Release Notes](https://github.com/MobilityData/gbfs/releases/tag/v3.1-RC)).

```suggestion
```